### PR TITLE
Parse speedrun.com run id if present in LS 1.6 file

### PR DESCRIPTION
**Pull request by BatedUrGonnaDie**

### CODE REVIEW

File: app/models/run.rb [hunk 1]
```diff
@@ -93,6 +93,7 @@ def parse(fast: true, convert: false)
         name: result[:name],
         program: result[:program],
         attempts: result[:attempts],
+        srdc_id: result[:srdc_id].blank? ? nil : result[:srdc_id],
```
Comment (glacials) [inline app/models/run.rb]: This can be simplified with `presence`: http://api.rubyonrails.org/v4.2.5/classes/Object.html#method-i-presence

Comment (BatedUrGonnaDie) [inline app/models/run.rb]: I did not realize that .present? would return false for a blank string, which was why I chose .blank?.  Will update accordingly.

Comment (glacials) [inline app/models/run.rb]: This is what I use to remember: http://stackoverflow.com/a/20663389/392225

Comment (BatedUrGonnaDie) [inline app/models/run.rb]: I can see that table being very useful.  Thanks for the link

**Pull request merged**